packages/microeditor-envelope/src/__tests__/AppFormerGwtApi.test.ts

```diff
@@ -0,0 +1,33 @@
+import { AppFormerGwtApi } from '../gwt/AppFormerGwtApi';
+import { DummyEditor } from './DummyEditor';
+import * as AppFormer from "appformer-js-core";
+import { GwtEditor } from '../gwt/GwtEditor';
+
+class DummyGWTEditor {
+  private wrappedEditor: AppFormer.Editor;
+
+  constructor (wrappedEditor: AppFormer.Editor) {
+    this.wrappedEditor = wrappedEditor;
+  }
+  public get(): AppFormer.Editor {
+    return this.wrappedEditor;
+  }
+}
```

Comment:
Are you using the Prettier formatter? I think the default is to have a blank line between methods..